tools/lint.py

```diff
@@ -118,9 +145,30 @@ def main():
     parser.add_argument("--files", nargs='*',
                         help="Lint these files or directories; "
                              "use **/*.py to lint all files")
+    parser.add_argument("--all", action='store_true',
+                        help="This overrides `--diff-against` and `--files` "
+                             "to lint all local files (excluding subprojects).")
+    parser.add_argument("--no-cython", dest='cython', action='store_false',
+                       help="Do not run cython-lint.")
```

Comment:
not sure yet whether/how to expose this via `dev.py`, but perhaps we can punt on that until the `spin` PR is in